### ISSUE TITLE
format of --help output needs adjusting

### ISSUE THREAD

**Issue by andy5995** [MegaGlest/megaglest-source#154]
I would like to offer to reformat the English language help output.

This is how it appears on my screen in a full screen terminal, and a 80x24 terminal.

[![Screenshot_2017-02-25_01-46-56.png](https://s32.postimg.org/jh8giyosl/Screenshot_2017_02_25_01_46_56.png)](https://postimg.org/image/7fd2otfk1/)

Before I start to make those changes, first I would like to know if I using the coding standards I used in #153 are acceptable. If not, I would still work on this issue.

Secondly, is reformatting the console output desired by the developers?

@titiger I know you would like to release soon; I could have this finished within the next 3 days.

First I think I would reformat the first couple pages, and then send an initial PR for review, so we could make sure everyone is happy. Then I would finalize it, and then send it for a final review and feedback. I would ask questions along the way if necessary.

**Comment by filux** [comment]
I don't have an opinion about does it need to be improved or not, but...
Don't forget that "man pages" are generated based on the help output (e.g. "man megaglest" to see manual for v3.12.0 in the modern version of distro/linux), so after any changes both screens should look better, not worse ;).

**Comment by andy5995** [comment]
@filux I didn't know MG had a man page! :) Where is it located? I tried `man megaglest` but I assume since I used the installer that the man page is not in the _man_ search path.

After more consideration, I see that I would need longer than three days to work on this and it would require more discussion. Point being, my goal would be to start after the next release, and finish before the release after that.

If I were going to reformat, I'd also want to change some of the actual text in the cases where it could be shortened.

For instance, this current excerpt:
```
	printf("\n%s=x\t\tLoads the last saved game.",GAME_ARGS[GAME_ARG_AUTOSTART_LAST_SAVED_GAME]);
	printf("\n                     \t\tWhere x is an optional name of the saved game file to load.");
	printf("\n                     \t\tIf x is not specified we load the last game that was saved.");
```

to help with reformatting, I would ask if it could be changed to:

```
	printf("\n%s=x\t\tLoads the last saved game, or loads [x] (name of saved game filename)",GAME_ARGS[GAME_ARG_AUTOSTART_LAST_SAVED_GAME]);
```

And perhaps I should be more specific when I talk about reformatting: Basically alignment. And I think i could shorten some of the paragraphs:

[![Screenshot_2017-02-25_12-02-53.png](https://s2.postimg.org/hymc9jz3t/Screenshot_2017_02_25_12_02_53.png)](https://postimg.org/image/ksphn019x/)

Here is an example of where I streamlined the output of the --auto-test option. Note I did suggest a change to one of the arguments of the option. In my mind, if 'z' can only be blank or 'exit', then 'exit' may be a better choice and more user-friendly.

[![Screenshot_2017-02-25_12-14-17.png](https://s1.postimg.org/9ghpngz6n/Screenshot_2017_02_25_12_14_17.png)](https://postimg.org/image/e2dtvtkpn/)
